tox.ini

```diff
@@ -42,3 +42,4 @@ commands =
 [flake8]
 max-line-length = 100
 exclude = silverstrike/migrations,silverstrike/views/__init__.py
+ignore = E226,E302,E41
```

Comment:
why did you add this?  
* E226 enforces whitespace around operators
* E302 enforces blank lines between top level elements in a file
* E41: I'm not sure what this actually is, I can't find anything to it online
Both are good things that I want to have.

Reply:
by reverting this change and actually fixing the output of flake instead of ignoring the warnings.

By adding a ignore you are removing the existing defaults of flake, so other errors are shown that were not shown in the past because I actually like the defaults